samples/subsys/usb/testusb/README.rst

```diff
@@ -0,0 +1,58 @@
+.. _testusb_sample:
+
+Testusb
+#######
+
+At the moment below is description for native_posix configuration only.
```

Comment:
Better would be:

    This sample and instructions are for testing USB with the native_posix configuration
    only and has not been tested with other board configurations.

Reply:
Actually this is not correct, this sample works also in other configurations, only instructions for this board configurations.